core/domain/exp_domain_test.py

```diff
@@ -37,10 +37,18 @@ def mock_get_filename_with_dimensions(filename, unused_exp_id):
         filename, 490, 120)
 
 
+# This function should be only be used while loading v26 textangular
+# exploration. If we do not use a mock there, the loading will
+# not pass the validation, since the current html validation
+# assumes CKEditor formatting.
 def mock_validate_rte_format(unused_html_list, unused_rte_format):
```

Comment:
Can you change the name to mock_validate_rte_format_for_v26?

Reply:
Done.